genologics/lims.py

```diff
@@ -394,16 +394,33 @@ def get_processes(self, last_modified=None, type=None,
         params.update(self._get_params_udf(udf=udf, udtname=udtname, udt=udt))
         return self._get_instances(Process, params=params)
 
-    def get_workflows(self):
+    def get_workflows(self, name=None):
         """Get the list of existing workflows on the system """
-        params = self._get_params()
+        params = self._get_params(name=name)
         return self._get_instances(Workflow, params=params)
 
-    def get_protocols(self):
+    def get_process_types(self, displayname=None):
+        """Get a list of process types with the specified name."""
+        params = self._get_params(displayname=displayname)
+        return self._get_instances(Processtype, params=params)
+
+    def get_reagent_types(self, name=None):
+        params = self._get_params(name=name)
+        return self._get_instances(ReagentType, params=params)
+
+    def get_protocols(self, name=None):
         """Get the list of existing protocols on the system """
-        params = self._get_params()
+        params = self._get_params(name=name)
         return self._get_instances(Protocol, params=params)
 
+    def get_reagent_kits(self, name=None):
+        params = self._get_params(name=name)
+        return self._get_instances(ReagentKit, params=params)
+
+    def get_reagent_lots(self, name=None, kitname=None, number=None):
+        params = self._get_params(name=name, kitname=kitname, number=number)
+        return self._get_instances(ReagentLot, params=params)
+
```

Comment:
```get_reagent_kits``` and  ```get_reagent_lots``` already exists with the same implementation.
Could you remove the one you added?